templates/missing-component.html.twig

```diff
@@ -5,3 +5,7 @@
  */
 #}
 {# @todo: Remove this template once the migration to ECL 2.0 is done. #}
+<div class="ecl-u-bg-grey-25 ecl-u-pa-s">
+  <p class="ecl-u-type-color-blue ecl-u-type-paragraph-s ecl-u-type-bold">Missing component '{{ theme_hook_original }} - {{ variant }}' template.</p>
+</div>
```

Comment:
Can we remove this please? It clutters the theme, we have this information in the error message anyway. If you really want to keep it for development reasons you can make it configurable with a [Drupal settings](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Site%21Settings.php/function/Settings%3A%3Aget/8.2.x) toggle and disable it by default. Up to you.

Reply:
Removed.